Sources/Protocols/ErrorLogger.swift

```diff
@@ -0,0 +1,7 @@
+//  ErrorLogging.swift
+
+import Foundation
+
+public protocol ErrorLogger {
+    func logError(_ error: ToggleManager.ProviderValueError)
```

Comment:
The signature of the method can be without “Error” in the name as the type of the parameter gives it away:

```
func log(_ error: ToggleError)
```